cmd/server.go

```diff
@@ -175,7 +175,10 @@ func getSelectedServer(ctx *cli.Context, cfg *config.Config) (string, error) {
 // prompt the user to select one.
 func serverFromInput(ctx *cli.Context, cf *config.Config) (string, error) {
 	servers := getServers(cf)
-	displayListServers(ctx, servers)
+	err := displayListServers(ctx, servers)
```

Comment:
This could be `if err := displayListServers(ctx, servers); err != nil ... ` ?

Reply:
I'm not a big fan of the one-line err checks, but if it's more idiomatic I'm ok with that: 5d03f2a :smile: